spec/features/users/edit_user_spec.rb

```diff
@@ -60,6 +60,7 @@
 
       expect do
         click_link_or_button 'Save'
+        sleep 0.1
```

Comment:
I'm not a big fan of this approach. If you set a  fixed`sleep`, either it's definitely long enough to fix the flake, in which case most of the time you're slowing down the test runs more than you need to, or it's short enough that sometimes it doesn't fix the sleep, in which case you still have a flaky test.

You're better off trying to use capybara's built-in solution for this, which is using a matcher that will only match the UI after the page reload has completed. Capybara will wait for that matcher to match something, meaning it will definitely wait long enough, but won't wait longer than necessary.

Reply:
yeah, I agree, it's not the most elegant, but it works,
for `password_unlock_spec.rb`